docker-compose.yml

```diff
@@ -81,19 +81,6 @@ services:
         reservations:
           memory: 512M
 
-  testrpc:
```

Comment:
oh niceee

Reply:
I can undo this, do we even use testrpc for anything? I do all my testing to rinkeby, just eats up ram if we're not using it actively.

Reply:
yeah lets ditch it if possible - but I want to make sure @octavioamu and @owocki don't use it either - can you put this in a separate PR?